napari/utils/_testsupport.py

```diff
@@ -317,6 +317,14 @@ def actual_factory(
 
     yield actual_factory
 
+    # Clear cached property `Action.injected`, see #7219 for details
+    # To be removed after ActionManager deprecation
+    from napari.utils.action_manager import action_manager
+
+    for action in action_manager._actions.values():
+        if 'injected' in action.__dict__:
+            del action.__dict__['injected']
```

Comment:
I think that we should move this to a separate fixture that should be autosued?
We may have a test that will interact with actions without using `make_napari_viewer`